controllers/cinder_controller.go

```diff
@@ -819,6 +823,27 @@ func (r *CinderReconciler) reconcileNormal(ctx context.Context, instance *cinder
 		return ctrl.Result{}, err
 	}
 
+	// create CronJob
+	cronjobDef := cinder.CronJob(instance, serviceLabels, serviceAnnotations)
+	cronjob := cronjob.NewCronJob(
+		cronjobDef,
+		5*time.Second,
```

Comment:
nit: Since this is not a job we need "right now" we can probably set a higher reconciliation timeout value.

Reply:
Sure, but what would that value be?

Reply:
Good question, I don't know :-P